corgi/tasks/brew.py

```diff
@@ -71,8 +81,8 @@ def slow_fetch_brew_build(build_id: int, save_product: bool = True, force_proces
 
     softwarebuild, created = SoftwareBuild.objects.get_or_create(
         build_id=component["build_meta"]["build_info"]["build_id"],
```

Comment:
This looks like it's relying on Brew's data format. Should we just use the task's argument here?
```suggestion
        build_id=build_id,
```

Reply:
That makes sense to me. Updated.